packages/jazz-tools/src/tools/coValues/inbox.ts

```diff
@@ -408,3 +506,14 @@ function getAccountIDfromSessionID(sessionID: SessionID) {
 
   return;
 }
+
+function waitForFullStreaming(coValue: CoValueCore) {
```

Comment:
Just an idea: this feels like a useful function that could live in cojson and be unit tested in isolation